.eslintrc

```diff
@@ -0,0 +1,97 @@
+{
+    "env": {
+        "commonjs": true,
+        "node": true,
+        "es2017": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "arrow-spacing": [
+            "warn",
+            {
+                "after": true,
+                "before": true
+            }
+        ],
+        "strict": "error",
+        "block-spacing": "warn",
+        "brace-style": [
+            "warn",
+            "1tbs"
+        ],
+        "camelcase": "warn",
+        "comma-spacing": [
+            "warn",
+            {
+                "after": true,
+                "before": false
+            }
+        ],
+        "comma-style": [
+            "warn",
+            "last"
+        ],
+        "eol-last": "error",
+        "eqeqeq": "warn",
+        "func-call-spacing": "warn",
+        "keyword-spacing": [
+            "error",
+            {
+                "after": true,
+                "before": true
+            }
+        ],
+        "linebreak-style": [
+            "warn",
+            "unix"
+        ],
+        "max-len": [
+            "warn",
+            {
+                "code": 120
+            }
+        ],
+        "prefer-const": "warn",
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": "warn",
+        "semi-style": [
+            "error",
+            "last"
+        ],
+        "space-before-blocks": "warn",
+        "spaced-comment": [
+            "warn",
+            "always"
+        ],
+        "no-trailing-spaces": "warn",
+        "no-multi-spaces": "warn",
```

Comment:
thought this would be good to add. https://eslint.org/docs/rules/no-multi-spaces

In the commit https://github.com/hazelcast/hazelcast-nodejs-client/pull/808/commits/bbe9a22ff5f7f3024ab493ce0400355aacb9269b there are some fixes regarding this as well.

Reply:
Yes, that's a good addition.